nightshade-webclients.yml

```diff
@@ -114,26 +102,83 @@
       notify:
         - restart nginx
 
-    - name: Config for OMERO.web plugins
-      become: yes
-      template:
-        src: templates/omero-web-config-for-webapps.j2
-        dest: "{{ omero_web_basedir }}/config/omero-web-config-for-webapps.omero"
-        owner: "root"
-        group: "root"
-        mode: "u=rw,go=r"
-      notify:
-        - restart omero-web
-
   vars:
-    omero_web_config_set: "{{ omero_web_config_set_for_group | combine(omero_web_config_set_for_host) }}"
-    omero_figure_release: "{{ omero_figure_release_override | default('4.0.2') }}"
-    omero_fpbioimage_release: "{{ omero_fpbioimage_release_override | default('0.3.0') }}"
-    omero_iviewer_release: "{{ omero_iviewer_release_override | default('0.8.0') }}"
-    omero_parade_release: "{{ omero_parade_release_override | default('0.1.2') }}"
-    omero_webtagging_autotag_release: "{{ omero_webtagging_autotag_release_override | default('3.0.2') }}"
-    omero_webtagging_tagsearch_release: "{{ omero_webtagging_tagsearch_release_override | default('3.0.3') }}"
-    omero_web_release_version: "{{ omero_web_release_version_override | default('5.5.1') }}"
-    nginx_version: 1.14.2
+    omero_web_config_set_for_playbook:
+      omero.web.nginx_server_extra_config:
+        - 'include /etc/nginx/conf.d-nested-includes/*.conf;'
+    omero_web_config_set: >-
+      {{
+        omero_web_config_set_for_playbook | combine(
+        (omero_web_config_set_for_group | default({})),
+        (omero_web_config_set_for_host | default({})))
+      }}
+
+    omero_web_release: "{{ omero_web_release_override | default('5.6.1') }}"
+    omero_figure_release: "{{ omero_figure_release_override | default('4.2.0') }}"
+    omero_fpbioimage_release: "{{ omero_fpbioimage_release_override | default('0.4.0') }}"
+    omero_iviewer_release: "{{ omero_iviewer_release_override | default('0.9.0') }}"
+    omero_parade_release: "{{ omero_parade_release_override | default('0.2.0') }}"
+    # omero_webtagging_autotag_release: "{{ omero_webtagging_autotag_release_override | default('3.0.2') }}"
```

Comment:
maybe okay for 3.1.0 now

Reply:
Tat will require a another deployment run, leave for the next PR?

Reply:
web tagging access.log hits for reference:
```
     80 2019-Jan
     95 2019-Feb
    127 2019-Mar
     81 2019-Apr
    169 2019-May
    499 2019-Jun
    207 2019-Jul
    164 2019-Aug
   1554 2019-Sep
    217 2019-Oct
    440 2019-Nov
     89 2019-Dec
     67 2020-Jan
```